packages/lab/src/navbar/NavbarItem.tsx

```diff
@@ -60,6 +60,7 @@ const NavbarItemRoot = styled.div`
   }
 
   &[data-active='true'] {
+    background-color: #2f394a;
```

Comment:
@devdesrel Can we use SD colors?

Reply:
@doniyor2109 I see custom color was used in NavBar too, so let's make a different PR for that